src/main/steps/c100-rebuild/contact-representative/content.ts

```diff
@@ -0,0 +1,56 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  serviceName: 'Child arrangements',
+  title: 'Contact your legal representative',
+  paragraph: 'If you have a legal representative and want them to complete the application for you:',
+  bulletPoints: [
+    'get in touch with your legal representative',
+    'ask them what information they need from you to complete the application',
+    'ask them to explain the next steps',
+  ],
+  warningText: {
+    text: 'Do not complete the application yourself if you plan to have a legal representative fill it in.',
+    iconFallbackText: 'Warning',
+  },
+});
+
+const cy = () => ({
+  serviceName: 'Child arrangements - welsh',
+  title: 'Contact your legal representative - welsh',
+  paragraph: 'If you have a legal representative and want them to complete the application for you: - welsh',
+  bulletPoints: [
+    'get in touch with your legal representative - welsh',
+    'ask them what information they need from you to complete the application - welsh',
+    'ask them to explain the next steps - welsh',
+  ],
+  warningText: {
+    text: 'Do not complete the application yourself if you plan to have a legal representative fill it in. - welsh',
+    iconFallbackText: 'Warning - welsh',
+  },
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {},
+  submit: {
+    text: () => 'Close the application',
```

Comment:
Can we move this to variables

Reply:
Moved inside variables, all tests passing ✅